Code/PgSQL/rdkit/Makefile

```diff
@@ -47,7 +47,7 @@ SHLIB_LINK += -pthread
 
 SHLIB_LINK += ${THREADLIBS}
 
-PG_CPPFLAGS = -I${BOOSTHOME}/include -I${RDKIT}/Code -DRDKITVER='"007300"' ${AVALONFLAGS} ${INCHIFLAGS} ${POPCOUNTFLAGS}
+PG_CPPFLAGS = -I${BOOSTHOME}/include -I${RDKIT}/Code -DRDKITVER='"007300"' ${AVALONFLAGS} ${INCHIFLAGS} -DUSE_BUILTIN_POPCOUNT ${POPCOUNTFLAGS}
```

Comment:
Right, when I noticed it was missing I didn't think about this.. it's a preprocessor flag, so the placement is not wrong, but maybe adding it to the ${POPCOUNTFLAGS} would have been cleaner?